pallets/ah-migrator/src/staking/bags_list.rs

```diff
@@ -121,8 +182,15 @@ impl<T: Config> crate::types::AhMigrationCheck for BagsListMigrator<T> {
 		// Assert storage "VoterList::ListBags::ah_post::correct"
 		// Assert storage "VoterList::ListBags::ah_post::consistent"
 		assert_eq!(
-			rc_pre_payload, ah_messages,
+			rc_pre_translated, ah_messages,
 			"Bags list data mismatch: Asset Hub data differs from original Relay Chain data"
 		);
+
+		// Run bags-list pallet integrity check
+		#[cfg(feature = "try-runtime")]
+		<pallet_bags_list::Pallet<T, pallet_bags_list::Instance1> as frame_election_provider_support::SortedListProvider<
```

Comment:
Note to self: Remove the feature gate in the SDK so that we can use it here as well.